### PR TITLE
Add support for pyenv-virtualenv

### PR DESCRIPTION
**Pull request by clbarnes**
If the user sets an environment variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+
+### oh-my-zsh
+
+```sh
+git clone https://github.com/davidparsson/zsh-pyenv-lazy.git ~/.oh-my-zsh/custom/plugins/pyenv-lazy
+```
+
+## Usage
+
+In your `~/.zshrc`, add `pyenv-lazy` to your `plugins` variable.
```
Comment (davidparsson) [inline README.md:18]: I think this is a part of the installation for `oh-my-zsh` as well, right?

**Review by davidparsson — APPROVED** [review]
Thanks for your PR! I've only got one comment but I can fix that myself.

File: README.md (hunk 1 above)
Comment (clbarnes) [reply README.md:18]: Yes, that may be oh-my-zsh specific. I'm new to zsh!

Comment (davidparsson) [reply README.md:18]: That's fine, I've fixed that on master. Thanks for your contribution!

**Pull request merged**